dev/cnf/resources/bnd/bundle.props

```diff
@@ -38,7 +38,7 @@ Bundle-Version: ${bFullVersion}.${if;${driver;gradle};${version.qualifier};eclip
 -consumer-policy :    ${range;[==,+)}
 
 # Require a 1.6 JDK (at least): see section 3.4.1 of OSGi 4.3 core spec
```

Comment:
comments should be updated here and in rejar.props because they explicitly refer to JDK 1.6

Reply:
I talked with Andy.  As I expressed in the other PR, the comment is referring specifically to OSGi requirements and we don't want this default ever set to something lower than 6.  7 is at least 6 which is what the comment is stating.